### ISSUE TITLE
The website is illegible due to low contrast between the text and the background

### ISSUE THREAD

**Issue by sompylasar** [sumanjs/sumanjs.github.io#2]
<img width="1422" alt="screen shot 2017-10-16 at 10 25 48 am" src="https://user-images.githubusercontent.com/498274/31625973-e19108b4-b25c-11e7-8465-fc332d88a3cd.png">

**Comment by ORESoftware** [comment]
Yeah, thanks, I haven't worked on the docs in 6 months - I can ping you in a month when I update them, until then, I would just hold out! I was thinking of using Gitbook instead of JSDoc - https://www.gitbook.com

**Comment by sompylasar** [comment]
Yes, GitBook seems to be mainstream now.

**Comment by ORESoftware** [comment]
Yeah, when I actually officially release Suman with finished docs, I will send out a one-time email to all the followers of the repo, so I can ping you then.
